CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 
 ### UI Improvements
 
+1. [17849](https://github.com/influxdata/influxdb/pull/17849): Move Organization navigation items to user menu.
 1. [17714](https://github.com/influxdata/influxdb/pull/17714): Cloud environments no longer render markdown images, for security reasons.
 1. [17321](https://github.com/influxdata/influxdb/pull/17321): Improve UI for sorting resources
 1. [17740](https://github.com/influxdata/influxdb/pull/17740): Add single-color color schemes for visualizations
```

Comment:
nit: i think we append to the bottom not the top. not that it's a very big deal

```suggestion
1. [17714](https://github.com/influxdata/influxdb/pull/17714): Cloud environments no longer render markdown images, for security reasons.
1. [17321](https://github.com/influxdata/influxdb/pull/17321): Improve UI for sorting resources
1. [17740](https://github.com/influxdata/influxdb/pull/17740): Add single-color color schemes for visualizations
1. [17849](https://github.com/influxdata/influxdb/pull/17849): Move Organization navigation items to user menu.
```

Reply:
Ya, here's the Quartz PR: https://github.com/influxdata/quartz/pull/2530